README.md

```diff
@@ -151,7 +151,7 @@ research using the following BibTeX:
                   Ziebarth, Malte and
                   Quinn, Jamie and
                   Wessel, Paul},
-  title        = {{PyGMT: A Python interface for the Generic Mapping Tools}},
+  title        = {PyGMT: A Python interface for the Generic Mapping Tools},
```

Comment:
The double curly brackets is set to preserve the proper capitalization of PyGMT (without it, some bibliography software might render it as Pygmt). See e.g. https://tex.stackexchange.com/questions/7288/preserving-capitalization-in-bibtex-titles and https://en.wikibooks.org/wiki/LaTeX/Bibliography_Management#Preserving_case_of_letters.

Reply:
Interesting! I was not aware of this issue. Redone in commit https://github.com/GenericMappingTools/pygmt/pull/3313/commits/9e868b5be1b062d0df37ba6627c36fab011bb83b.

Reply:
One of those things you learn when a copyeditor goes through your BibTeX citations one by one and points out all the mistakes https://github.com/weiji14/deepbedmap/commit/f86b366f5758e96d8ae0ebe8f791420242546f4d :slightly_smiling_smile: